packages/web/src/pages/purchases-and-sales/SalesPage.tsx

```diff
@@ -133,11 +143,39 @@ const RenderSalesPage = () => {
   const isEmpty = status === Status.SUCCESS && sales.length === 0
   const isLoading = statusIsNotFinalized(status)
 
+  const downloadCSV = useCallback(async () => {
```

Comment:
I'm wondering if we want any kind of loading state while we are waiting for the CSV to be generated and returned? Otherwise the user might click the button mulitple times because it appeared to do nothing.
If we do want to do that, it might be better to put the logic into a separate hook that can manage the state.

Reply:
Interesting... I think no loading state as far as spec goes, but also probably we just didn't think about it. Something to bring up